tutorial/index.jade

```diff
@@ -108,88 +89,82 @@
+      was successful and if so, set User's `verified` field to true.
 
-      That's all for token verification! However, our verification form wouldn't
-      be very usable if there wasn't a way to resend a verification code if the
-      message didn't arrive at the end user's handset for whatever reason.
+      That's all for token verification! However, our verification form wouldn't be very
+      usable if there wasn't a way to resend a verification code if the message
+      didn't arrive at the end user's handset.
 
       ---
 
       **See Also:**
       * [Java Client for Authy API](//github.com/authy/authy-java)
 
-  .step(
-    data-title='Display User Data',
-    data-file='src/main/java/com/twilio/verification/servlet/AccountServlet.java',
-    data-highlight='28-39')
+  .step(data-title='Re-sending The Code', data-file='src/main/java/com/twilio/verification/servlet/ResendTokenServlet.java', data-highlight='36-46')
     :markdown
-      ## Display User Data
+      ## Re-sending the Code
 
-      This servlet will provide the necessary information to display the user's
-      data.
+      Since the form for re-sending the code is one line see the
+      [Verification form](//github.com/TwilioDevEd/account-verification-servlets/blob/master/src/main/webapp/verifyCode.jsp#L28-L40).
+      We're going to skip that for this tutorial. Let's just look at the servlet.
 
-      This should suffice for confirmation in the browser that the user has been
-      verified. Let's see how we might send that text message next.
+      This method loads the `user` associated with the request and then
+      uses the same Authy API method we used earlier to resend the code.
 
-  .step(
-    data-title='Sending the Confirmation Message',
-    data-file='src/main/java/com/twilio/verification/lib/Sender.java',
-    data-highlight='24-41')
+      To wrap things up, let's implement the last step where we confirm that the
+      user's account has been verified with a text message and a success page.
+
+  .step(data-title='Sending the Confirmation Message', data-file='src/main/java/com/twilio/verification/lib/Sender.java', data-highlight='25,28,34-41')
     :markdown
       ## Sending the Confirmation Message
 
-      Much as we did for our Authy client, we create a single instance of the
-      Twilio REST API helper, called `client` in this example.
-
-      Then all we need to do to is send an SMS to the user's phone. This is done
-      using the `send()` method. Notice that we'll use the user's `fullNumber()` to
-      make sure we support international numbers. The `getFullPhoneNumber()` method
-      in the `User` model simply returns a combination of the `countryCode` and
-      `phoneNumber` that the user provided upon registration.
+      We create a single instance of the Twilio REST API helper,
+      called `client` in this example.
 
+      Then we need get the account, get the messageFactory and finally use its `sendMessage` method
```

Comment:
Would be better if we use something like "..., get the `messageFactory` method and ...". What do you think?